scanpy_scripts/cmd_options.py

```diff
@@ -1650,6 +1667,171 @@
             'connectivity value of 1)'
         ),
     ],
+    
+    'scrublet': [
+        *COMMON_OPTIONS['input'],
+        *COMMON_OPTIONS['output'],
+        click.option(
+            '--input-obj-sim', 'adata_sim',
+            type=click.Path(exists=True, dir_okay=False),
+            default=None,
+            help='(Advanced use case) Optional annData object generated by '
+            'sc.external.pp.scrublet_simulate_doublets(), with same number of  '
+            'vars as adata. This should have been built from input_obj after '
+            'filtering genes and cells and selcting highly-variable genes.'
+        ),
+        click.option(
+            '--threshold',
+            type=click.FLOAT,
+            default=None,
+            show_default=True,
+            help='Doublet score threshold for calling a transcriptome a '
+            'doublet. If not set, this is set automatically by looking for the '
+            'minimum between the two modes of the doublet_scores_sim_ histogram. '
+            'It is best practice to check the threshold visually using the '
+            'doublet_scores_sim_ histogram and/or based on co-localization of '
+            'predicted doublets in a 2-D embedding.'
+        ),
+        *COMMON_OPTIONS['scrublet'],
+        click.option(
+            '--expected-doublet-rate',
+            type=click.FLOAT,
+            default=0.05,
```

Comment:
If we leave these defaults to None, won't scanpy use its own internal defaults? It already happened to Andrey on Seurat that because we keep this values, at somepoint we deviate from the actual tool default (on a version change) and then our cli default behaviour is not the intended one by developers at that version of the software.

Reply:
Fair point, though I think there's some debate to be had there. Like you say though, probably something for a package-wide overhaul.